static/workspace.html

```diff
@@ -332,6 +332,28 @@ <h3 class="display-5" id="workspace-title">Untitled</h3>
             </shadow>
           </value>
         </block>
+        <block type="plotting_two_values">
+          <value name="xValue">
+            <shadow type="math_number">
+              <field name="NUM">0</field>
```

Comment:
Why do you want to use default values at this place? :D I need a quick reminder

Reply:
It makes checking if the block (and associated features) work much faster. It also allows a user to just quickly throw the block into the workspace and see how it behaves.

Reply:
But its confusing for the user imo :/

Reply:
That is possible. I had the loop-blocks in mind when designing the block optics, where the same style is used.  We could perhaps remove the default values from the "y-Value" input, because that is the one field that will be overridden everytime.

Reply:
Sounds great! 👍